packages/account/src/Components/currency-selector/currency-selector.tsx

```diff
@@ -218,7 +218,6 @@ const CurrencySelector = observer(
                                             <React.Fragment>
                                                 <RadioButtonGroup
                                                     className='currency-selector__radio-group currency-selector__radio-group--with-margin'
-                                                    label={localize('Cryptocurrencies')}
```

Comment:
Should we pass` is_title_enabled = false` since else header element is  is still rendered in DOM?
<img width="646" alt="Screenshot 2024-05-22 at 4 19 05 PM" src="https://github.com/binary-com/deriv-app/assets/84661147/79106d88-44bd-473a-91eb-792c5e8b1cf2">

Reply:
@amina-deriv in my opinion would be better to change the logic inside this component, make `label` optional, and write like this:

`{label && <h2>...</h2>}`

But in the current implementation, of course, we need to pass `is_title_enabled={false}`, agree with you

Reply:
@amina-deriv @sergei-deriv Since we are not using this component at many places, and as far as I checked it is in the currency selector file only I will be removing the `is_title_enabled` , as the label is optional already I will use label instead of is_title_enabled, what you say?

Reply:
@mitra-deriv if you look at the components code, you'll see is_title_enabled is true by default. In result, we will have an empty `h2` tag in the DOM. Just pass `is_title_enabled={false}` for `RadioButtonGroup` and it's OK

Reply:
@mitra-deriv but if you really want to change the component, then ok :) It's up to you))) I'll approve any solution